src/rgw/CMakeLists.txt

```diff
@@ -128,6 +128,8 @@ set(rgw_a_srcs
   rgw_crypt_sanitize.cc
   rgw_iam_policy.cc)
 
+list(APPEND rgw_a_srcs async/throttle.cc)
+
```

Comment:
still not sure where in the tree this stuff should live. none of it is specific to rgw, so maybe under common instead?  i'd eventually like to use the async::Completion for the librados bindings in librados/librados_asio.h

Reply:
The throttler code as such doesn't seem to have anything rgw specific so should be fine moving under common::async as well